pages.fr/windows/cd.md

```diff
@@ -0,0 +1,33 @@
+# cd
+
+> Affiche le répertoire de travail actuel ou permet de se déplacer vers un autre répertoire.
+> Dans PowerShell, cette commande est un alias de `Set-Location`. Cette documentation est basée sur la version `cmd` de `cd`.
+> Plus d'informations : <https://learn.microsoft.com/windows-server/administration/windows-commands/cd>.
+
+- Voir la documentation de la commande PowerShell équivalente :
```

Comment:
```suggestion
- Affiche la documentation de la commande PowerShell équivalente :
```

Please use the third person singular present indicative tense.

Ref: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#french-specific-rules